Added Good Folder structure of Project

RoshanAnsy opened the pull request:
I added two **new folder** 
1. DataHouse :-> It contains all data of our projects (**All Jason data**)
2. controller :-> Now it has all the logic code of our project. If someone wants to add more functionality to this project they have to write the **logic code** in this folder so that the code separation follows.
3. I renamed the router file name now it becomes **productsRoutes** and also removed all logic code from this file to the controller folder.

Now code readability has **increased** due to code **sepresection**.

VighneshManjrekar reviewed (changes requested):
just pull the latest commit and change the directory structure do not add additional code and packages. And rename the DataHouse to `data`. Remove .env file

RoshanAnsy commented:
I changed 

deleted:    .env
        deleted:    DataHouse/Products.json
        deleted:    DataHouse/Users.json
        modified:   controller/products.js
        modified:   package-lock.json
        modified:   package.json
        modified:   server.js

RoshanAnsy reviewed (commented):
done

VighneshManjrekar commented on server.js:
```diff
@@ -1,9 +1,13 @@
 
 app.get("/", (req, res) => res.send("Hello World!"));
+app.use(express.json());
+// app.use(bodyParser.json());
```
remove this comment

VighneshManjrekar commented on package.json:
```diff
@@ -5,13 +5,16 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "cookie-parser": "^1.4.6",
```
no need of cookie-parser package

scripts: start and dev doing the same thing instead include nodemon script there

install nodemon as an dev dependency

VighneshManjrekar commented on controller/products.js:
```diff
@@ -0,0 +1,29 @@
+exports.getAllProduct=async(req,res)=>{
+    try{
+
+        // const {minPrice,maxPrice}=req.body
```
min max price filtering issue isn't yet resolved remove that code. Pull the latest commit and don't include additional lines of code just copy paste the code by making right directories

RoshanAnsy replied:
is the code is not working for min max price ??

you mean I need to remove the commented line from the code??

VighneshManjrekar replied:
yes your PR is still under review for min max price product don't include that code

RoshanAnsy commented:
Now I think everything has done please check it.

VighneshManjrekar commented on routes/productRoutes.js:
file name must be `routes/products.js` same as `controllers/products.js` to maintain uniformity

RoshanAnsy commented:
changed 

![Screenshot 2023-10-17 122610](https://github.com/CSI-DMCE-2023/CSI-API-HacktoberFest-2023/assets/91673102/4d984bd6-7310-4810-8e33-35576c05a4df)

VighneshManjrekar reviewed (changes requested):
Pull latest commit locally and then implement this folder structure. Everything is good to go but code is outdated.

VighneshManjrekar commented:
Closing this PR since no reply from @RoshanAnsy

RoshanAnsy commented:
what kind of changes do you want can you explain it ?
